pepper-apis/src/main/java/org/broadinstitute/ddp/export/DataExportCoordinator.java

```diff
@@ -84,6 +84,11 @@ public boolean export(StudyDto studyDto) {
         }
 
         if (csvBucket != null) {
+            withAPIsTxn(handle -> {
+                exporter.computeMaxInstancesSeen(handle, activities);
+                exporter.computeActivityAttributesSeen(handle, activities);
```

Comment:
Only the CSV exports leverages these two pieces of data, so we only compute them during CSV export to save costs.